gridt/src/app/home/home.page.html

```diff
@@ -59,9 +59,9 @@ <h3>
                         </h3>
                         <p>{{ isLeaderDone(leader, movement) ? 'Done!' : 'Pending...' }}</p>
                       </div>
-                      <small *ngIf="isLeaderDone(leader, movement) && leader.last_signal.message">
-                        {{ leader.last_signal.message }}
-                      </small>
+                      <div *ngIf="isLeaderDone(leader, movement) && leader.last_signal.message" id="expandable">
+                        <signal-message signalText={{leader.last_signal.message}}></signal-message>
```

Comment:
This element has only one real input and it does not necessarily need to be named, so what you could strive for (unless you have a good counter argument) is:

`<signal-message>{{ leader.last_signal.message }}</signal-message>`

Reply:
that is what i had initially, but then i needed to switch over all of  the functionality for the switching the view to the signal message component. If i were to do it like this i will have to replace the <p> in the signal message component with <ng-content> and i am not sure the styling would work with that as good as it does now.

Reply:
I feel like something went wrong when you typed this message. I also don't think I completely understand the argument, would you care to elaborate?